ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -297,6 +298,24 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView, TokenizedCont
         self.leadingView = leadingView
     }
 
+    /// - Parameters:
+    ///   - style: The `TableViewHeaderFooterView.Style` used to set up the view.
+    ///   - title: The title string.
+    ///   - backgroundColor: The background color of the `backgroundView`
```

Comment:
Would be good to get input @sophialee0416 on this parameter's naming and documentation. i.e. should it be `customBackgroundColor`? And what do we want to put in the comment to help inform the consumer when it makes sense to diverge from the default background color and how to do so without sacrificing on "fluent-ness" of design